addons/medical/functions/fnc_onMedicationUsage.sqf

```diff
@@ -77,8 +85,16 @@ _decreaseAmount = 1 / _timeInSystem;
 _viscosityAdjustment = _viscosityChange / _timeInSystem;
 
 [{
-    params ["_args", "_idPFH"];
-    _args params ["_target", "_timeInSystem", "_variable", "_amountDecreased","_decreaseAmount", "_usedMeds", "_viscosityAdjustment", "_painReduce"];
+    private ["_args", "_target", "_timeInSystem", "_variable", "_amountDecreased","_decreaseAmount", "_usedMeds", "_viscosityAdjustment", "_painReduce"];
+    _args = _this select 0;
+    _target = _args select 0;
+    _timeInSystem = _args select 1;
+    _variable = _args select 2;
+    _amountDecreased = _args select 3;
+    _decreaseAmount = _args select 4;
+    _viscosityAdjustment = _args select 5;
+    _painReduce = _args select 6;
```

Comment:
Also bug from the clean up. Another reason to merge this and redo it in small parts instead